tiny edits to upgrades (#1696)
timescale/docs: cloud/service-operations/maintenance.md

[TimescaleDB release notes][timescale-relnotes].

<highlight type="warning">

> Your Timescale Cloud service will be unavailable until the upgrade is complete. 

Reviewer: ```suggestion
Your Timescale Cloud service is unavailable until the upgrade is complete. 
```

> This can take several hours. We estimate roughly one second of downtime per 

Reviewer: ```suggestion
This can take several hours. To estimate the length of time, it is usually one second of downtime per 
```

> 100MB, but strongly encourage testing on a fork first for a better estimate.

Reviewer: ```suggestion
100&nbsp;MB, but for a better estimate, you can test on a fork first.
```

</highlight>

### Recommended practices for upgrading

> We strongly recommend the following for a smooth upgrade experience:

Reviewer: ```suggestion
For a smooth upgrade experience, make sure you:
```

> *   Plan ahead! Upgrades will cause downtime, so ideally perform an upgrade 

Reviewer: ```suggestion
*   Plan ahead. Upgrades cause downtime, so ideally perform an upgrade 
```
